### PR TITLE
Custom schema

### PR DESCRIPTION
**Pull request by zenhack**

### CODE REVIEW

File: tests/unit/moc/rest.py [hunk 1]
```diff
@@ -174,6 +176,28 @@ def request(self):
...
+            return json.dumps(x * y * z)
+
+    def api_call(self):
+        return json.dumps(14)
+
+    def request(self):
+        return wsgi_mkenv('POST', '/product',
+                            data=json.dumps({'x': 2, 'y': 7}))
+
+
```
Comment (gsilvis) [inline tests/unit/moc/rest.py]: Testing some invalid calls would be good

**Comment by gsilvis** [comment]
Everything else here looks good to me.  +1

**Comment by lokI8** [comment]
All looks good outside the invalid calls. contingent +1

**Comment by zenhack** [comment]
@gsilvis, @lokI8, I added a test. Look good?

**Comment by gsilvis** [comment]
All right, that looks good to me. No need to test every way a schema can fail, that's the job of the schema library.  full +1 now.

**Comment by zenhack** [comment]
Conflicts were pretty trivial, merging after verbal +1's.

**Pull request merged**